zndraw/zndraw.py

```diff
@@ -199,6 +199,15 @@ def __delitem__(self, index: int | slice | list[int]):
             retries=self.timeout["call_retries"],
         )
 
+    def _repr_html_(self):
+        from IPython.display import IFrame
+
+        address = f"{self.url}/token/{self.token}"
+        # TODO: save address and do not replace in post_init
+        address = address.replace("ws", "http")
+        log.info(f"Opening ZnDraw at {address}")
+        return IFrame(address, width="70%", height=600)._repr_html_()
```

Comment:
Could you add:

https://github.com/zincware/ZnDraw/blob/2da4c5a4cc9946ccfc14fafd30d89db9a3bae4e8/zndraw/zndraw.py#L63-L66

```python
from typing import TypedDict

class JupyterConfig(TypedDict):
    width: str|int
    height: str|int

class ZnDraw: ...

    jupyter_config: JupyterConfig|None = None
```

and set the default values in the `__post_init__`?